### PR TITLE
Add endpoint for prometheus metrics

### PR DESCRIPTION
**Pull request by worldtiki**
Prometheus metrics can be scraped from /prometheus
Also fixed a small TODO for the duration of the polls done by the Kafka consumer

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <aws-java-sdk.version>1.11.516</aws-java-sdk.version>
     <zipkin-reporter.version>2.8.1</zipkin-reporter.version>
     <spring.version>2.1.3.RELEASE</spring.version>
+    <micrometer.version>1.1.0</micrometer.version>
```
Comment (nikos912000) [inline pom.xml]: Not sure if this is relevant, but Micrometer versions >1.1.1 include fixes for Kafka consumer metrics: https://github.com/micrometer-metrics/micrometer/releases

Comment (worldtiki) [reply pom.xml]: Thanks. Updated to the latest.

**Comment by adriancole** [comment]
wonder how hard it would be to implement the same or similar prometheus metrics as zipkin collector does...

**Comment by worldtiki** [comment]
> wonder how hard it would be to implement the same or similar prometheus metrics as zipkin collector does...

Assuming you're talking about these https://github.com/openzipkin/zipkin/tree/master/zipkin-server#collector

Yeap, it would make sense for them to be similar. I think I can work on this in a few days.

Thanks for the feedback!

**Comment by worldtiki** [comment]
I'm merging this pr now.
I'll add the metrics for spans processed, dropped, etc. later this week.
I'll try to follow a similar format as Zipkin.

**Pull request merged**